server/public/model/user.go

```diff
@@ -62,6 +62,8 @@ const (
 	UserLocaleMaxLength   = 5
 	UserTimezoneMaxRunes  = 256
 	UserRolesMaxLength    = 256
+
+	DesktopTokenTTL = time.Minute * 3
```

Comment:
I'd rather not have it configurable at this point since users may set it way too long or too short by mistake and cause issues with their system. I think 3 minutes is a good TTL for these tokens, but we could change that in the future if we have a case to make it configurable.